Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs

```diff
@@ -9,24 +9,44 @@ public class receiveSignalLockedDoor : receiveSignal {
 	private Collider2D collider;
 
     public switchScene doorToNextLevel;
+    public string opensOrClosesUponTrigger = "opens";
```

Comment:
Yeah Ill change it, couldn't come up with a good variable name for a boolean lol